src/resolvers/teams.js

```diff
@@ -97,20 +97,38 @@ const teamsResolvers = {
       return true;
     },
 
-    async updateTeam(root, { id, name, organization, amountRaised, itemsSold, isArchived }) {
-      const team = await models.Team.findOne({ where: { id } });
-      if (team === null) {
-        throw new Error('Team Not Found');
+    async updateTeamName(root, { id, name }) {
+      console.log(id, name);
+      try{
+        const team = await models.Team.update({name}, {
+          where: { id },
+          returning: true
+        });
+        if(team.length == 2){
+          return team[1][0].dataValues;
+        }
       }
-      team.name = name;
-      team.organization = organization;
-      team.amountRaised = amountRaised;
-      team.itemsSold = itemsSold;
-      team.isArchived = isArchived;
+      catch(error){
+        console.log(error);
```

Comment:
I think you want to throw the error again here, otherwise I'm not sure what the resolver actually returns — but you want the resolver to return an error status code by throwing